app/serializers/merchant_serializer.rb

```diff
@@ -1,8 +1,4 @@
 class MerchantSerializer
   include FastJsonapi::ObjectSerializer
   attributes :name, :id
-
-  attributes :revenue do |merchant|
-    merchant.attributes[:revenue]
-  end
```

Comment:
Remove revenue because we dont want revenue to display everytime Merchant is called on